sslio.c

```diff
@@ -87,8 +87,8 @@ void redisFreeSsl(redisSsl *ssl){
     free(ssl);
 }
 
-int redisSslCreate(redisContext *c, const char *capath, const char *certpath,
-                   const char *keypath, const char *servername) {
+int redisSslCreate(redisContext *c, const char *caPath, const char *certPath,
```

Comment:
Any specific reason why these were changed to capitalize `Path` in each variable name?

Reply:
I follow the camelCase conventions in the header file (https://github.com/redis/hiredis/blob/master/sslio.h) so I changed it to capitalize.